src/iexamine.cpp

```diff
@@ -3431,6 +3432,144 @@ void iexamine::climb_down( player &p, const tripoint &examp )
 }
 
 /**
+ * Use base command to edit options and utalize base functions.
+ * 
+ * TODO: Finish writting. Change to uimenu.
+ */
+void iexamine::base_control( player &p, const tripoint &examp) {
+    #define base (p->home) 
+    if (base == NULL){
+        add_msg("The GM prevents you from messing with other people's base settings.");
+        return;
+    }
+    
+    uimenu desktop;
+    desktop.text = _("Base Command Options:");
+    desktop.addentry( 1, true, MENU_AUTOASSIGN, _("Ration Supplies.") );
+    desktop.addentry( 2, true, MENU_AUTOASSIGN, _("Ammunition Type Prefrences.") );
+    desktop.addentry( 3, true, MENU_AUTOASSIGN, _("Crafting Priorities.") );
+    if (base.get_level() > 1){//add lv.2 options
+        desktop.addentry( 4, true, MENU_AUTOASSIGN, _("View Status.") );
+        desktop.addentry( 5, true, MENU_AUTOASSIGN, _("Base Planner.") )
+        if (base.get_level() > 2){//add lv.3 options
+            desktop.addentry( 6, true, MENU_AUTOASSIGN, _("Personel Managment.") );
```

Comment:
All of those things are perfectly mundane. They shouldn't need a computer. Only things like machinery settings and automatic security should require computers.

Reply:
I'll change them.  The levels were kinda arbitrarily assigned anyways.

Reply:
I feel that there should be some recognition that while it's possible to do most things with pen/paper a lot of stuff just won't be worth the effort for a survivor without the help of a computer;  You can make detailed construction plans on paper but it takes orders of magnitude more time than with CAD software.  Maybe saying basic stuff like designating areas (crafting, sleeping etc./ define area to guard) would be for pen/paper but construction plans require software?

Reply:
Leave limiting for much later. First get it to work completely with no limits on player's options, then break it up into levels in further PR.